uber/templates/macros.html

```diff
@@ -119,13 +119,13 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, copy_address=False, name_prefix="") -%}
     {% set group_or_attendee=group if group else attendee %}
     {% set id_prefix=group.id|replace("-","") if group else attendee.id|replace("-","") %}
-    {% if not is_prereg_dealer %}
-        <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
-        <script type="text/javascript">{% include "region_opts.html" %}</script>
-    {% endif %}
+
+    <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
```

Comment:
I think this will double-print these sources if c.COLLECT_FULL_ADDRESS is turned on (that is, if the macro is called twice in a page).

You said you were getting missing script errors, so I think double-printing is the better option here and fixing that is low priority compared to making everything actually work.